src/Overview/components/ProductInfo.jsx

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
 import StarRating from '/src/Widgets/StarRating.jsx'
+import {FaFacebookSquare} from 'react-icons/fa'
+import {FaTwitterSquare} from 'react-icons/fa'
+import {FaPinterest} from 'react-icons/fa'
 
 const ProductInfo = ({product}) => {
+
   return (
     <div className='overview-widget-productInfo'>
       <div>
-       <div><StarRating/> <p>See all reviews</p></div>
+       <div className='stars-reviews'><StarRating/> <p className='review'>See all reviews</p></div>
```

Comment:
What is the reason why we have a class name declaration on a <p>?  would a <h1-6> work better here?